src/transformers/modeling_utils.py

```diff
@@ -2809,13 +2801,10 @@ def from_pretrained(
             quantization_method_from_config = config.quantization_config.get(
                 "quant_method", QuantizationMethod.BITS_AND_BYTES
             )
-        if quantization_method_from_config is not None and quantization_method_from_args is not None:
-            if quantization_method_from_config != quantization_method_from_args:
-                raise ValueError(
-                    f"The model is already quantized with {quantization_method_from_config}. "
-                    f"You can't quantize it again with {quantization_method_from_args}"
-                )
-        if quantization_method_from_config == QuantizationMethod.GPTQ and quantization_method_from_args is not None:
+        if (
+            quantization_method_from_config in (QuantizationMethod.GPTQ, QuantizationMethod.AWQ)
+            and quantization_method_from_args is not None
+        ):
             loading_attr_dict = quantization_config.get_loading_attributes()
             for attr, val in loading_attr_dict.items():
```

Comment:
I see that `get_loading_attributes` is not defined in `AWQConfig` . If it is not defined, we won't be able to overwrite the quantization config stored in config, hence performing the fusing. It is strange that it worked.

Reply:
Hmm indeed it was not defined before, I just added it now